.env.example

```
@@ -1,3 +1,2 @@
 ASTRO_DB_REMOTE_URL=
-ASTRO_DB_APP_TOKEN=
-ASTRO_DATABASE_FILE=
\ No newline at end of file
+ASTRO_DB_APP_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)